chia/_tests/environments/wallet.py

```diff
@@ -247,9 +247,9 @@ async def change_balances(self, update_dictionary: Dict[Union[int, str], Dict[st
 
     async def wait_for_transactions_to_settle(
         self, full_node_api: FullNodeSimulator, _exclude_from_mempool_check: List[bytes32] = []
-    ) -> List[TransactionRecord]:
+    ) -> List[LightTransactionRecord]:
         # Gather all pending transactions
-        pending_txs: List[TransactionRecord] = await self.wallet_state_manager.tx_store.get_all_unconfirmed()
+        pending_txs: List[LightTransactionRecord] = await self.wallet_state_manager.tx_store.get_all_unconfirmed()
```

Comment:
Can we add a separate `get_all_unconfirmed_light` so changes like this aren't necessary?  I'm worried about having two different types causing a lot of headache upstream a la `Program`/`SerializedProgram`.